.github/actions/docker-build-image/action.yml

```diff
@@ -209,3 +209,11 @@ runs:
         labels: ${{ steps.meta.outputs.labels }}
         push: true
         provenance: false
+
+    - name: Print tags
+      if: env.PUSH_IMAGE == 'true'
+      shell: bash
+      run: |
+        echo quay.io/alfresco/${{ env.IMAGE_NAME }}
```

Comment:
why not print them in summary instead? (like on https://github.com/Alfresco/alfresco-build-tools/blob/65df92980d9506697dd4da49104d7d597bf27c2c/.github/actions/maven-build-and-tag/action.yml#L179)

Reply:
<img width="1133" alt="Screenshot 2024-06-18 at 13 28 16" src="https://github.com/Alfresco/alfresco-build-tools/assets/208200/14392d74-887c-4591-afd4-a4a343fe675f">
Thanks for a tip